### ISSUE TITLE
Web UI pagination uses count of event object-attribute collections, not attributes

### ISSUE THREAD

**Issue by chrisinmtown** [MISP/MISP#6893]
The web UI seems to implement pagination based on the number of object-attribute collections, not the number of attributes within them, which can result in enormous pages and slow load times.

We are running MISP version 2.4.133 and starting to use object templates to create collections of attributes within a named object-attribute on events. 

- Severe case: our dev server has an event with 2047 attributes spread across 7 objects.  The web page loaded for that event is enormous - it contains all 2047 attributes, no pagination is offered.   
- Difficult but manageable case:   an event with 676494 attributes spread across 75730 object-attribute collections.  Each web UI page shows 60 objects, and with approx 10 attr/obj the resulting page is very large. 
- Compare either page mentioned above with an event that has plain attributes (no object-attribute collections), for which the event page shows the first 60 attributes, and shows pagination controls like "next", "previous", "view all" as well as a link for page 1, 2, 3, ....

I expect that defining & implementing pagination for object-attributes based on the number of attributes (not just the number of collections) is non-trivial, but it's necessary.

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | RedHat
| OS version (client)       | all
| PHP version               | 7.4
| MISP version / git hash   | 2.4.133
| Browser                   | all

### Expected behavior

An event page shows a manageable number of object-attribute collections based on the total number of attributes, and allows pagination forwards/back.

### Actual behavior

Up to 60 object-attribute collections are shown, regardless of the total number of attributes that brings, which may be thousands.

### Steps to reproduce the behavior

Define an event, add object-attribute collections, add attributes to each collection, and view the event in the web UI.

**Comment by chrisinmtown** [comment]
Possible dupe of https://github.com/MISP/MISP/issues/6287

**Comment by chrisinmtown** [comment]
Is it possible that the web UI is paginating on the OBJECT COUNT, without regard to the number of attributes in them?  Because in other events that use object-attribute collections I do see pagination being done.  Hmm I better update the title here.

**Comment by JakubOnderka** [comment]
You should try to update to latest version. It will not solve your problem completely, but at least generating and loading events with a lot of attributes is much faster now. 

> Is it possible that the web UI is paginating on the OBJECT COUNT, without regard to the number of attributes in them? 

Exactly.

**Comment by chrisinmtown** [comment]
Thanks @JakubOnderka  for confirming my suspicion.  That pagination approach is too simplistic for the data that we throw at MISP.

**Comment by chrisinmtown** [comment]
@PROTechThor please remove the "support" tag and add a "bug" or whatever tag.  I don't need support here. I understand how the web UI works now and I assert that the behavior is deficient, needs to be revised.

**Comment by iglocska** [comment]
Indeed, this is a limitation of the current approach that was built with smaller events in mind (also it explains the horrid performance with large events). I've started working on an alternate implementation for large events, where loose attributes and objects are paginated in two separate lists. Just need to find the time to get back to it.

**Comment by github-germ** [comment]
I guess this was never accomplished?
